### PR TITLE
Enable appveyor

### PR DESCRIPTION
**Pull request by Eldinnie**
I made a new bot in my account and hid the relevant information in appveyors environment vars.
I modified appveyor.yaml to use the new tests and codecov.

If all is well this should mean I've enabled appveyor

### CODE REVIEW

File: tests/test_meta.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def call_pre_commit_hook(hook_id):
 
 @pytest.mark.nocoverage
 @pytest.mark.parametrize('hook_id', argvalues=('yapf', 'flake8', 'pylint'))
-@pytest.mark.skipif(not os.getenv('TRAVIS'), reason='Not running in travis.')
+@pytest.mark.skipif(not (os.getenv('TRAVIS') or os.getenv('APPVEYOR')), reason='Not running in travis.')
```
Comment (bomjacob) [inline tests/test_meta.py]: Could you edit the reason too?

File: tests/test_bot.py [hunk 2]
```diff
@@ -364,6 +368,7 @@ def test_get_chat_member(self, bot, channel_id):
         assert chat_member.status == 'administrator'
         assert chat_member.user.username == 'EchteEldin'
 
+    @pytest.mark.skipif(os.getenv('APPVEYOR'), reason='Skip game tests on Appveyor')
```
Comment (bomjacob) [inline tests/test_bot.py]: Maybe we should write why we're skipping and not just that we are in the reason?
(I know we haven't really done this before but it makes more sense to do it like that I think)
(applies elsewhere)

**Comment by bomjacob** [comment]
Mostly looks good! Good call on just doing it instead of waiting for us to set up more bots.
I left a few notes in my review :)

**Comment by Eldinnie** [comment]
So, As far as I'm concerned this is the first step towards adding Appveyor to our CI cycle. For now I'm skipping tests for telegram-communication regarding stickers and games because they're not added on the bot (yet).
I'm also skipping tests for jobqueue and messagequeue because they rely on timing to a measure more precise then Appveyor can provide.

Coverage increased because now the branches that split for windows and posix systems now both are executed.

I addressed @bomjacob 's review comments.

**Comment by Eldinnie** [comment]
closes #591

**Pull request merged**